lib/logger/lib/logger.ex

```diff
@@ -99,6 +99,22 @@ defmodule Logger do
   `:crash_reason`, `:initial_call`, and `:registered_name` are available
   only inside behaviours such as GenServer, Supervisor, and others.
 
+  For example, you might wish to include a custom error code in your logs:
```

Comment:
I think it's important to stress that this is **metadata**, otherwise I'm afraid the reader could think that the "error code" is a specific functionality.

```suggestion
  For example, you might wish to include a custom error code metadata in your logs:
```

Reply:
What if it read like this:
```
  For example, you might wish to include a custom `:error_code` bit of metadata in your logs:
```
?